.env-example-files/.env-example

```diff
@@ -0,0 +1,3 @@
+# hardhat forking only works with mainnet right now
+FORKING_URL=https://eth-mainnet.alchemyapi.io/v2/<YOUR_ALCHEMY_API_KEY>
+BLOCK_NUMBER=11712717
```

Comment:
nothing specific about the block. we can choose any block where the dai and adai addresses contain enough dai and adai at the time of the forking block.

but we really do need to pin it. otherwise we will get flaky tests.